core/lib/src/main/java/org/opennms/core/utils/TimeSeries.java

```diff
@@ -27,6 +27,20 @@
         public String getName() {
             return m_name;
         }
+
+        public String getDescr() {
+            switch (m_name) {
```

Comment:
This switch can be avoided by passing the description directly in the constructor i.e.:

    RRD(RRD_TIME_SERIES_STRATEGY_NAME, "RRD (RRDTool or JRobin)"),